tests/ml/ml_test.py

```diff
@@ -37,6 +44,39 @@ def test_valid_sklearn_pca(df_iris):
     np.testing.assert_almost_equal(ds_pca.evaluate('PCA_0'), sklearn_trans[:, 0])
 
 
+@pytest.mark.parametrize("n_components", [5, 15, 150])
+@pytest.mark.parametrize("matrix_type", ['gaussian', 'sparse'])
+def test_random_projections(n_components, matrix_type):
+    df = vaex.from_dict(data=data_maker(n_rows=100_000, n_cols=31))
+    features = df.get_column_names()
+
+    rand_proj = vaex.ml.RandomProjections(features=features, n_components=n_components, matrix_type=matrix_type)
+    df_trans = rand_proj.fit_transform(df)
+    assert np.array(rand_proj.random_matrix_).shape == (n_components, 31)
+    assert len(df_trans.get_column_names(regex='^rand')) == n_components
```

Comment:
should we test the fast api? or since that is automated, we assume it works?

Reply:
Ok I can re-write it to use the fast api